alby-nostr-wallet-connect/umbrel-app.yml

```diff
@@ -0,0 +1,32 @@
+manifestVersion: 1
+id: alby-nostr-wallet-connect
+name: Nostr Wallet Connect
+tagline: The power of the zaps at the tip of your fingers.
+icon: https://raw.githubusercontent.com/getAlby/umbrel-community-app-store/master/alby-nostr-wallet-connect/images/nwc_icon.svg
+category: Finance
+version: "0.1.0"
+port: 58000
+description: >-
+  Connect your LND node over Nostr.
+
+  Zap directly from within supported apps like Damus & Amethyst.
+
+  Amethyst step-by-step guide: https://blog.getalby.com/native-zapping-in-amethyst
+
+  Damus (currently only on Testflight) video guide: https://streamable.com/bareih
+developer: Alby
+website: http://getalby.com/
+submitter: kiwiidb
+submission: https://github.com/getAlby/nostr-wallet-connect
+repo: https://github.com/getAlby/nostr-wallet-connect
+support: https://github.com/getAlby/nostr-wallet-connect/issues
+gallery:
+  - https://raw.githubusercontent.com/getAlby/umbrel-community-app-store/master/alby-nostr-wallet-connect/images/nwc_1.png
+  - https://raw.githubusercontent.com/getAlby/umbrel-community-app-store/master/alby-nostr-wallet-connect/images/nwc_2.png
+  - https://raw.githubusercontent.com/getAlby/umbrel-community-app-store/master/alby-nostr-wallet-connect/images/nwc_3.png
+releaseNotes: >-
+  - Budget & time-limited session support
+dependencies: []
```

Comment:
Since this depends on the Lightning app (LND), which in turn depends on the Bitcoin app, we can add those apps as dependencies so that a user must install Lightning and Bitcoin prior to installing Alby NWC.

```suggestion
dependencies:
  - bitcoin
  - lightning
```

They will then see this in the app store:

<img width="425" alt="image" src="https://github.com/getumbrel/umbrel-apps/assets/85373263/43332677-f183-4da2-8b81-00746d59a1a1">

and if they try to install anyways, but are missing a dependency they will be shown this:

<img width="416" alt="image" src="https://github.com/getumbrel/umbrel-apps/assets/85373263/72675f24-f07b-4a4e-9462-8e9d668a84c9">

Reply:
Ok. If I may ask, what happened to being able to run LND with Neutrino on Umbrel? Would be nice to get up and running faster without running Bitcoin Core.